src/components/atoms/Navigator/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { memo } from "react";
+
+import { useT } from "@reearth/i18n";
+import { styled } from "@reearth/theme";
+
+import Icon from "../Icon";
+
+import { useNavigator } from "./hooks";
+import { CompassAngle } from "./types";
+
+export type Props = {
+  degree: number;
+  /**
+   * Pass degree of circle as callback arguments.
+   * This event is invoked when outer circle is clicked and mouse move.
+   * If click is dismissed then this event is stopped.
+   */
+  onRotate?: (degree: number) => void;
+  /**
+   * Pass distance of movement as x,y axis(px) and degree of circle as callback arguments.
+   * This event is invoked when inner circle button is clicked and mouse move.
+   * If click is dismissed then this event is stopped.
+   */
+  onOrbit?: (angle: CompassAngle) => void;
+  /**
+   * This event is invoked when center of circle is clicked.
+   */
+  onRestoreRotate?: () => void;
+  /**
+   * Pass + or -  as callback arguments.
+   */
+  onClickHelp?: () => void;
+  onZoomIn?: () => void;
+  onZoomOut?: () => void;
+};
+
+const Navigator: React.FC<Props> = memo(function NavigatorPresenter({
+  degree,
+  onRotate,
+  onOrbit,
+  onRestoreRotate,
+  onClickHelp,
+  onZoomIn,
+  onZoomOut,
+}) {
+  const {
+    compassRef,
+    compassDegree,
+    compassFocusAngle,
+    isMovingAngle,
+    handleOnMouseDownAngle,
+    handleOnMouseDownCompass,
+  } = useNavigator({ degree, onRotate, onOrbit });
+  const t = useT();
+  return (
+    <Container>
+      <CompassContainer>
+        <Compass ref={compassRef}>
+          <>
```

Comment:
Do you need this fragment? 
I've seen other developers use it where I feel its not needed, so I am curious if this is just a mistake or you are using it purposefully.

Reply:
I fixed!
[refactor: remove fragment](https://github.com/reearth/reearth-web/pull/308/commits/8e645e54eef3f9d5fa4269b34f9219cea30de9a0)